src/components/Stack/Stack.stories.tsx

```diff
@@ -86,7 +86,7 @@ export default {
       options: ['wrap', 'nowrap', 'wrap-reverse'],
       control: { type: 'radio' },
     },
-    gap: {
+    flexGap: {
```

Comment:
This would be a breaking change requiring a major release. Would it be better to leave the prop as is and change the functionality only?